### PR TITLE
Added po/pot files for change password and forgot password

### PR DESCRIPTION
**Pull request by madhavrathi**

### CODE REVIEW

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 1]
```diff
@@ -82,11 +83,11 @@ export default class ChangePassword extends Component {
               let result = zxcvbn(newPassword);
               state.newPasswordScore = result.score;
               let strength = [
-                'Worst',
-                'Bad',
-                'Weak',
-                'Good',
-                'Strong'
+                <Translate key={1} text="Worst" />,
```
Comment (rishiraj824) [inline src/components/Auth/ChangePassword/ChangePassword.react.js:86]: no need for this, please let it be as it is,

File: src/components/Auth/ChangePassword/ChangePassword.react.js [hunk 2]
```diff
@@ -116,23 +117,23 @@ export default class ChangePassword extends Component {
 
         if (this.state.passwordError && event.target.name === 'password') {
             this.emailErrorMessage = '';
-            this.passwordErrorMessage = 'Minimum 6 characters required';
+            this.passwordErrorMessage = `${<Translate text="Minimum 6 characters required" />}`;
```
Comment (rishiraj824) [inline src/components/Auth/ChangePassword/ChangePassword.react.js]: don't make changes here, its not required

File: src/components/Auth/ForgotPassword/ForgotPassword.react.js [hunk 3]
```diff
@@ -254,7 +254,7 @@ class ForgotPassword extends Component {
 						open={true}
 						onRequestClose={this.handleClose}
 					>
-						<Translate text={this.state.msg}/>
+						{this.state.msg}
```
Comment (rishiraj824) [inline src/components/Auth/ForgotPassword/ForgotPassword.react.js]: please don't remove the <Translate /> component

File: src/locale/de/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -194,12 +194,55 @@ msgstr "Chat anonym"
...
+
+msgid "Good"
+msgstr "Gut"
+
+msgid "Strong"
+msgstr "Stark"
+
+msgid "Minimum 6 characters required"
+msgstr "Mindestens 6 Zeichen erforderlich"
+>>>>>>> Added po/pot files for change password and forgot password
```
Comment (rishiraj824) [inline src/locale/de/LC_MESSAGES/messages.po]: please check this line

File: src/locale/messages.pot [hunk 5]
```diff
@@ -199,6 +199,52 @@ msgstr ""
...
+msgstr ""
+
+msgid "Invalid Credentials. Please check your Email or Password."
+msgstr ""
+
+msgid "Please login again."
+msgstr ""
+
+msgid "Please Try Again."
+>>>>>>> Added po/pot files for change password and forgot password
```
Comment (rishiraj824) [inline src/locale/messages.pot]: please check this line

**Review by rishiraj824 — COMMENTED** [review]
please don't change the component files. The <Translate /> component should only be added inside render() function

File: src/components/Auth/ChangePassword/ChangePassword.react.js (hunk 1 above)
Comment (madhavrathi) [reply src/components/Auth/ChangePassword/ChangePassword.react.js:86]: This is required as they needed to be translated. I have checked the other approach, they did not show results, so we have to stick with this Please suggest if you have any alternative

**Review by rishiraj824 — APPROVED** [review]
thanks

**Pull request merged**